schemas/schema-schema.ipldsch

```diff
@@ -148,30 +148,131 @@ type AnyScalar union {
 	| Float float
 } representation kinded
 
+## AdvancedDataLayout holds `advanced` definitions which currently only require
+## a name.
+##
+## Use `advanced Foo` rather than `type Foo` to indicate an advanced data layout
+## (ADL) with that name which can be used as a representation for type
+## definitions whose kind the ADL is able to support.
+##
+## The ADL name is currently the only identifier that can be used to make a
+## connection with the algorithm/logic behind this ADL. Future iterations may
+## formalize this connection by some other means.
+##
+type AdvancedDataLayout struct {
```

Comment:
Might be a naming bikeshed, and feel free to dismiss, but I might like to try sneaking one more word into the name of this one -- it took me a second to sort out whether these was the block with the details, or the much smaller thing thing that refers to it.

(I came up with `ADLBlock` and `ADLRef` in my scratchpad, but I'm not attached to those (nor even particularly pleased with that much acronym).)